test/integrationTests/features/reportIssue.test.ts

```diff
@@ -3,15 +3,15 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { getFakeVsCode } from '../testAssets/fakes';
+import { getFakeVsCode } from '../../unitTests/testAssets/fakes';
 import reportIssue from '../../../src/shared/reportIssue';
 import { expect } from 'chai';
 import { vscode } from '../../../src/vscodeAdapter';
 import { Options } from '../../../src/shared/options';
-import { FakeMonoResolver, fakeMonoInfo } from '../fakes/fakeMonoResolver';
-import { FakeDotnetResolver } from '../fakes/fakeDotnetResolver';
+import { FakeMonoResolver, fakeMonoInfo } from '../../unitTests/fakes/fakeMonoResolver';
+import { FakeDotnetResolver } from '../../unitTests/fakes/fakeDotnetResolver';
 import { DotnetInfo } from '../../../src/shared/utils/getDotnetInfo';
```

Comment:
This file is already faking a bunch of objects and does not use the real getDotnetInfo. Should I just define the object here and move this file back to unit test?